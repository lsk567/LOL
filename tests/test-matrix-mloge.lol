Matrix t = Matrix([10.0,20.0]);
mloge(t);

printm(t);
