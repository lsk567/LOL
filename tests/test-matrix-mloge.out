||	2.30259	2.99573	||

